src/main/java/barcus/task/Event.java

```diff
@@ -0,0 +1,63 @@
+package barcus.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Task with from and to dates
+ */
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+
+    private final static DateTimeFormatter fromFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm");
+    private final static DateTimeFormatter toFormatter = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
```

Comment:
This code looks to be repeated in the Deadline class as well, could we try to "shift" it up to its parent class and handle formatting there instead?